app/src/main/java/org/sopt/and/ui/components/MypageScreen/MyPageProfileSection2.kt

```diff
@@ -0,0 +1,40 @@
+package org.sopt.and.ui.components.MypageScreen
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun MyPageProfileSection2(
+    sectionDescription: String,
+    connectedUrl: String            /*  나중에 화면 전환할 때 쓰이지 않을까 싶어서 ....*/
+){
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .background(Color.DarkGray)
+            .padding(vertical = 1.dp)
+    ) {
+        Column(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(15.dp)
+        ) {
+            Text(
+                "$sectionDescription",
+                color = Color.Gray
+            )
+            Text(
+                "구매하기 >",
+                color = Color.White
+            )
```

Comment:
named call argument 가 없다면 가독성에서도 아쉬워지고, 추후 수정을 할 때에도 불편할 수 있어요. 붙이는 습관을 들이면 좋을 것 같아요!
```suggestion
            Text(
                text = "$sectionDescription",
                color = Color.Gray
            )
            Text(
                text = "구매하기 >",
                color = Color.White
            )
```

Reply:
아하 named argument에 대한 이야기가 여기 이미 달렸네용